ui/component/or-attribute-input/src/index.ts

```diff
@@ -133,19 +144,59 @@ export function getAttributeValueTemplate(
                 if (inputType === InputType.JSON && value !== null && typeof(value) !== "string") {
                     value = JSON.stringify(value, null, 2);
                 }
-                return Util.getAttributeValue(attribute, attributeDescriptor) ? Util.getAttributeValue(attribute, attributeDescriptor) : value;
+                currentValue = Util.getAttributeValue(attribute, attributeDescriptor) ? Util.getAttributeValue(attribute, attributeDescriptor) : value;
+                return currentValue;
+            };
+
+            // todo: using this fn to put a value into current value so the button knows about it is a bit of a detour but it works for now
+            const setHelperTextChanged = (event: any) => {
+                currentValue = event.path[0].value;
```

Comment:
This feels like sorcery. I can't seem to find a way to fetch the current input value in another way though.